### DISCUSSION TITLE
Left drawer isn't sticky and being scrolled with content/body

### DISCUSSION THREAD

**Discussion by Guzmank90** [quasarframework/quasar#10381] [General - Components / Directives / etc] [answered]
Hello,
I'm having an issue with left drawer specificaly, while right one is sticky, I need the left one to be sticky as well to fullfill the design.
Here's a fiddle - https://jsfiddle.net/5dyt1ghv/1/

**Comment by ontwikkelfabriek** [comment, marked as answer]
@Guzmank90 

This is a Quasar specific _need to know_ thing about the layout. To make the left part sticky, change the `l` to a capitalized `L` ( just like R):

```javascript
 <q-layout view="hHh LpR fFf">
```

The layout builder will explain a lot:
https://quasar.dev/layout-builder

**Reply by Guzmank90** [reply]
Worked like charm, thanks!
